modules/ossm-installing-the-istioctl-tool.adoc

```diff
@@ -6,29 +6,54 @@
 [id="ossm-installing-the-istioctl-tool_{context}"]
 = Installing the Istioctl tool
 
+Install the `istioctl` configuration command line utility to debug and diagnose {istio} service mesh deployments.
+
 .Prerequisites
 
 * You have access to the {ocp-product-title} web console.
 
+* The {SMProduct} 3 Operator is installed and running.
+
+* You have created at least one `Istio` resource.
+
 .Procedure
 
-. In the {ocp-product-title} web console, click the question mark icon and select *Command Line Tools*.
+. Confirm which version of the `Istio` resource runs on the installation by running the following command:
++
+[source,terminal]
+----
+$ oc get istio -ojsonpath="{range .items[*]}{.spec.version}{'\n'}{end}" | sed s/^v// | sort
+----
++
+If there are multiple `Istio` resources with different versions, choose the latest version. The latest version is displayed last.
+
+. In the {ocp-product-title} web console, click the Help icon and select *Command Line Tools*.
 
 . Click *Download istioctl*. Choose the version and architecture that matches your system.
+  
+* link:https://mirror.openshift.com/pub/cgw/servicemesh/latest/istioctl-1.24.4-linux-amd64.tar.gz[Linux (x86_64, amd64)]
```

Comment:
Would it be possible to use `{istio-latest}` attribute for having dynamic version URLs? 

```suggestion
* link:https://mirror.openshift.com/pub/cgw/servicemesh/latest/istioctl-{istio-latest}-linux-amd64.tar.gz[Linux (x86_64, amd64)]
```

Reply:
I made this change and created a preview build. Unfortunately, the logic in the tool chain does not appear to be able to resolve this and it results in a broken link.

Reply:
That's a shame, it means we may have to update these links every new releases... For OSSM 3.0.2, it would already use `1.24.5` version